### ISSUE TITLE
BTC Values are now rounded to 5 decimal points - requesting a way to turn off rounding for Bitcoin account balances

### ISSUE THREAD

**Issue by geoff604** [LedgerHQ/ledger-live-desktop#4896]
#### Ledger Live Version and Operating System

- tested on Ledger Live  Ledger Live 2.40.2
- Platform and version: Windows 10

#### Expected behavior

I should be able to see all 8 decimal places of my BTC balance.

#### Actual behavior

As of version Ledger Live 2.40.2, bitcoin balances are now rounded to 5 digits everywhere in the application.

#### Steps to reproduce the behavior

Open Ledger Live 2.40.2 for an account which has at least 0.1 BTC. You will see that bitcoin balances are only shown as 0.1xxxx and the remaining digits are omitted.

It seems that rounding was enabled in the following change:
https://github.com/LedgerHQ/ledger-live-desktop/pull/4758

Preferably, there should be a way to disable rounding per-currency or to customize the rounding behavior to at least show all the BTC decimal points.

**Comment by Al-cm** [comment]
Same rounding cutoff issue applies for ETH balances.

**Comment by Steuberke** [comment]
Is there already a fix available? Its so annoying.

**Comment by bouchepat** [comment]
Yes, I agree, the rounding is annoying, please offer a setting for decimal points. Preferably a per coin-based setting.

**Comment by Al-cm** [comment]
Until this rounding customisation is implemented, I will use Electrum with Ledger for seeing exactly what btc amount I have in my wallet. Electrum can only be used for bitcoin, and not for other coins. And it is not a solution, but a stop-gap method for the rounding balance problem Ledger Live has at the moment.
